common/include/common/common_utils.hpp

```diff
@@ -237,7 +237,7 @@ static inline std::vector<blas1_param_t> get_blas1_params(Args& args) {
   if (args.csv_param.empty()) {
     warning_no_csv();
     std::vector<blas1_param_t> blas1_default;
-    for (index_t size = 4096; size <= 1048576; size *= 2) {
+    for (index_t size = 1048576; size <= 32 * 1048576; size *= 2) {
```

Comment:
This brings up the number of generated sizes to only 5, is this what we want or it doesn't matter?

Reply:
I updated the ranges. Now we run 8 different samples, and Google Benchmark will run multiple iterations for each sample until statistically significant results are obtained.